src/sql/workbench/contrib/connection/browser/media/connectionViewlet.css

```diff
@@ -1,138 +0,0 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
This file and all these resources weren't being used anywhere, they're pretty much duplicates of connectionViewletPanel.css so seemed to just be a missed part of a refactor.